package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "api.kitchen.support",
+  "version": "0.0.1",
+  "description": "Kitchen Support's RESTful API backend.",
+  "author": "Garrick Buckley <gbuckle@purdue.edu>",
+  "contributors": [
+    {
+      "name": "Garrick Buckley",
+      "email": "gbuckle@purdue.edu"
+    },
+    {
+      "name": "Jack Horton",
+      "email": "horton7@purdue.edu"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kitchensupport/api.git"
+  },
+  "dependencies": {
+    "express": "*",
+    "browserify": "*",
+    "gulp-ruby-sass": "*",
```

Comment:
1) gulp-sass is based on node-sass which is based on libsass, the C++ port of the Sass language. it is significantly faster than ruby-sass
2) indeed, gulp/gulp plugins and browserify are only needed for development and testing. technically they build the production version, but you wouldnt actually be building the production version in production. the only libraries needed for production in this commit are express and possibly `del`, though I dont know what `del` is for.